update_copyrights

```diff
@@ -120,30 +124,32 @@ for root, _, files in os.walk(options.repo):
             else:
                 unchanged.append(csv_data[file])
 
+malformed.sort(key=itemgetter(1))
 added.sort(key=itemgetter(1))
 changed.sort(key=itemgetter(1))
 incomplete.sort(key=itemgetter(1))
 update.sort(key=itemgetter(1))
 unchanged.sort(key=itemgetter(1))
 
-final_output = added + changed + incomplete + update + unchanged
+final_output = malformed + added + changed + incomplete + update + unchanged
 
 if options.output != "":
     with open(options.output, 'w') as f:
-        f.write("Date,File,License,Author - Real Name(other name);Real Name(other name);etc,Notes,Needs Update,MD5\n")
-        for row in final_output:
-            f.write(",".join(row)+"\n")
+        writer = csv.writer(f, lineterminator="\n")
+        writer.writerow(["Date", "File", "License", "Author - Real Name(other name);Real Name(other name);etc", "Notes", "Needs Update", "MD5"])
+        writer.writerows(final_output)
 else:
-    for row in final_output:
-        print(",".join(row))
+    writer = csv.writer(sys.stdout, lineterminator="\n")
+    writer.writerows(final_output)
 
 any_check_failed = False
 if len(removed) > 0:
     any_check_failed = True
     print("There are "+str(len(removed))+" removed images")
 
-if len(added) > 0 or len(changed) > 0 or len(incomplete) > 0 or len(update) > 0:
+if len(malformed) > 0 or len(added) > 0 or len(changed) > 0 or len(incomplete) > 0 or len(update) > 0:
     any_check_failed = True
+    print("There are "+str(len(malformed))+" rows with too many or too few fields, possibly due to an unquoted comma")
```

Comment:
My gut feeling is that this should go between "incomplete" and "update", rather than at the top.

Reply:
My gut feeling was that malformed rows that can't be parsed correctly (and therefore might also be changed, incomplete, or outdated, because the script might be misreading one field as another) are more severe than those with field values that are parsable but simply in need of updates.